client/package.json

```diff
@@ -28,7 +28,9 @@
   "categories": [
     "Programming Languages"
   ],
-  "activationEvents": [],
+  "activationEvents": [
```

Comment:
Have you also found a way to disable activation when .conf and .inc files are open?